app/src/main/res/values/strings.xml

```diff
@@ -135,9 +135,9 @@
     <string name="settings_download_path_title">Download path</string>
     <string name="settings_download_path_message">Files will be saved to %1$s</string>
     <string name="settings_hide_app_title">Hide the Magisk app</string>
-    <string name="settings_hide_app_summary">Install a proxy app with random package ID and custom app label</string>
+    <string name="settings_hide_app_summary">Install a proxy app with a random app ID and custom app label</string>
```

Comment:
Technically "application ID"(?)

Reply:
No. package is more accurate. Stop changing things you know nothing about.

Reply:
I thought it was "ApplicationId" instead of what I now think is "PackageName".
Helps to be able to look things up.